internal/executor/rpc/rpc.go

```diff
@@ -251,7 +268,7 @@ func (r *RPC) StreamLogs(stream api.CirrusCIService_StreamLogsServer) error {
 			currentTaskName = task.Name
 			currentCommand = x.Key.CommandName
 
-			streamLogger = r.logger.Scoped(task.UniqueName()).Scoped(currentCommand)
+			streamLogger := r.getCommandLogger(task, currentCommand)
```

Comment:
If you look at the `ReportSingleCommand()`, it already retrieves and validates the existence of the command.

Perhaps we should do the same here, which would allow us greatly simplify the `getCommandLogger()` since it'll only accept a `build.Command`.

Reply:
See 056773220f540420ab2b120da9101db961d1bae2